rero_ils/modules/ill_requests/views.py

```diff
@@ -35,16 +35,16 @@
 blueprint = Blueprint(
     'ill_requests',
     __name__,
-    url_prefix='/ill_requests',
+    url_prefix='/<string:viewcode>',
     template_folder='templates',
     static_folder='static',
 )
 
 
-@blueprint.route('/create/', methods=['GET', 'POST'])
+@blueprint.route('/ill_requests/create/', methods=['GET', 'POST'])
```

Comment:
The route `<string:viewcode>/ill_requests/create` seems very strange REST url.
Maybe better :
  * `ill_requests/create?viewcode=xxxx`
  * `ill_requests/<viewcode>/create``

I suggest the first one (seems better REST route). And test if `request.args.get('viewcode')` exists into the method (or extract this argument into a decorator)

Reply:
The structure of the url on the site always has as 2nd parameter in the url, the view code.